src/smolvm/cli/main.py

```
@@ -937,8 +982,18 @@ def _run_setup(parser: argparse.ArgumentParser, args: argparse.Namespace) -> int
         skip_deps=args.skip_deps,
         runtime_user=args.runtime_user,
         remove_runtime_config=args.remove_runtime_config,
+        for_bake=args.for_bake,
+        skip_kvm_check=args.skip_kvm_check,
+        skip_runtime_check=args.skip_runtime_check,
+        firecracker_version=args.firecracker_version,
     )
 
+    if options.for_bake:
+        console_stdout().print(
+            "[yellow]ℹ️  --for-bake skips KVM and runtime self-tests. "
```

Ruff (0.15.10)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)